apps/demo/stories/TapToEdit.stories.tsx

```diff
@@ -99,10 +74,6 @@ export const TapStory = (): ReactElement => {
         }}
       />
       <TapToEdit
-        key="bool"
-        name="bool"
-        openApiField="staffRoles.SuperUser"
```

Comment:
Ohh this was definitely me testing against our API :) probably best to remove, good call